From 395bb4cb37612a4ac94346531b13ff0e901ed05e Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Tue, 5 May 2015 23:59:17 +0200
Subject: [PATCH 1/2] Makefile: don't clone/build c-periphery

This patch adjusts lua-periphery Makefile to not clone and build the
c-periphery library by itself, and instead use the already available
c-periphery library.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lua-periphery/Makefile b/lua-periphery/Makefile
index bed9b32..a4c68eb 100644
--- a/lua-periphery/Makefile
+++ b/lua-periphery/Makefile
@@ -36,8 +36,8 @@ install:
 
 ###########################################################################
 
-$(LIB): $(C_PERIPHERY_LIB) $(SRCS)
-	$(CC) $(CFLAGS) $(LDFLAGS) $(SRCS) $(C_PERIPHERY_LIB) -o $@
+$(LIB): $(SRCS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(SRCS) -lc-periphery -o $@
 
 $(C_PERIPHERY_LIB): $(C_PERIPHERY)/Makefile
 	cd $(C_PERIPHERY); $(MAKE)
-- 
2.1.0

